airflow/utils/db.py

```diff
@@ -240,6 +240,23 @@ def create_default_connections(session=None):
         ),
         session
     )
+    merge_conn(
+        Connection(
+            conn_id="facebook_default",
+            conn_type="facebook_social",
+            schema="""
+            {
+                "facebook_ads_client": {
+                    "account_id": "act_123456789",
+                    "app_id": "1234567890",
+                    "app_secret": "1f45tghxxxx12345",
+                    "access_token": "ABcdEfghiJKlmnoxxyz"
+                }
+            }
+            """,
```

Comment:
I am not sure if we should keep dummy values here like that. Can we do this likewise gcp? `schema="default"`

Reply:
Is there any better way to show the schema ?? Cus for GCP there is a tutorial on google's website. Integrating with airflow. Any thoughts?